### DISCUSSION TITLE
unable to import pdf in zettlr

### DISCUSSION THREAD

**Discussion by WombatBeutler** [Zettlr/Zettlr#4094] [Q&A]
Hey there,
I'm trying to import a pdf.file in zettlr. I installed pandoc and LaTex and there is no problem to export a md.file as pdf, docx or anything else, It's also possible to import a docx-file, which will automatically be convert into a md.file. But every time I try to import a pdf (via file, import file) there is an error which says the filetype is unknown. 

![grafik](https://user-images.githubusercontent.com/84860661/214918513-48ac595f-0b7b-4651-928b-830acf2fd6a1.png)

I  have no idea why this happens :-(  and would be very grateful for any help!

**Comment by nathanlesage** [comment]
It is not possible to import PDF files. The PDF format has been invented mainly for display purposes, i.e., to print out something. The PDF format is basically “read only”. There are ways of getting text out of PDFs, but this task falls under the category of OCR (optical character recognition). There are tools out there that allow you to extract text from PDFs, but this is not within the scope of Zettlr. Zettlr only allows you to import editable text formats, such as Word documents, OpenOffice documents, and the likes.

**Reply by WombatBeutler** [reply]
<thanks!!! :-)
